src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -5104,6 +5104,26 @@
      * A triple dot for variable-length parameters. This token only ever occurs
      * in a parameter declaration immediately after the type of the parameter.
      *
+     * <p>For example:</p>
+     * <pre>
+     *  public void myShape(int... dimension) {
+     *
+     *  }
+     * </pre>
+     *
+     * <p>parses as:</p>
+     * <pre>
+     * |--LPAREN -&gt; (
```

Comment:
Please show ast from METHOD_DEF node.
We show all tokens that are shown in code example

Reply:
done, can you please re-review?